contracts/integrations/test/fuzz_tests/pool_membership_test.ts

```diff
@@ -22,14 +23,14 @@ export class PoolMembershipSimulation extends Simulation {
 
         const poolManagement = new PoolManagementSimulation(this.environment);
 
-        const actions = [
-            ...makers.map(maker => maker.simulationActions.validJoinStakingPool),
-            ...takers.map(taker => taker.simulationActions.validFillOrder),
-            poolManagement.generator,
-        ];
+        const [actions, weights] = _.unzip([
+            ...makers.map(maker => [maker.simulationActions.validJoinStakingPool, 0.2 / makers.length]),
```

Comment:
Could be good to make these constants to add context around `0.2` and `0.6` below.